pkg/arvo/app/graph-store.hoon

```diff
@@ -792,9 +816,37 @@
     =*  validator  i.t.wire
     =/  =rave:clay  [%next %b [%da now.bowl] /[validator]]
     [%pass wire %arvo %c %warp our.bowl [%home `rave]]~
+  ::
+      [%try-rejoin @ *]
+    =/  rid=resource:store  (de-path:res t.t.wire)
+    =/  nack-count    (slav %ud i.t.wire)
+    ?>  ?=([%b %wake *] sign-arvo)
+    ~?  ?=(^ error.sign-arvo)
+      "behn errored in backoff timers, continuing anyway"
+    =/  new=^wire  [%try-rejoin (scot %ud +(nack-count)) t.t.wire]
+    :_  this
+    [%pass new %agent [entity.rid %graph-push-hook] %watch resource+t.t.wire]~
   ==
 ::
-++  on-agent  on-agent:def
+++  on-agent
+  |=  [=wire =sign:agent:gall]
+  ^-  (quip card _this)
+  ?.  ?=([%try-rejoin @ *] wire)
+    (on-agent:def wire sign)
+  ?.  ?=(%watch-ack -.sign)
+    [~ this]
```

Comment:
this should crash, so that the subscription automatically dies when it receives it's first `%fact`

Reply:
I just realized I meant to do a %leave when it gets the watch-ack, that should be sufficient right?